infrastructure/afoco_source_bundle/proxy/conf.d/afoco.conf

```diff
@@ -0,0 +1,32 @@
+upstream strapi {
+    server cms:1337;
+}
+
+upstream client {
+    server client:3000;
+}
+
+server {
+    listen      80;
+
+    # client
+    location / {
+      proxy_pass http://client;
+    }
+
+    # CMS
+   location /cms/ {
```

Comment:
yeah, I understand your concern and from tech point of view and to prevent collisions, it makes sense to have cms behind `/cms/` path. Still feel like user would prefer url without it so they can just type `/admin` and not `/cms/admin` :). I believe that 6 special CMS routes required by Strapi is probably final and still manageable, but lets keep it under /cms prefix for now and we will see...

Reply:
I copy-pasted this setup into orcasa, and immediately upon login in into the admin and opening one of the sections, I started getting a bunch of 404 that were ultimately due to them not being mapped here. Also, next-api (a lib the FE team uses often) creates a bunch of stuff under /api, so that will come to bite us pretty soon